src/main/resources/db/migration/V1_70__create_grant_mandatory_question_table.sql

```diff
@@ -0,0 +1,66 @@
+-- grant_mandatory_questions
+CREATE TABLE IF NOT EXISTS public.grant_mandatory_questions (
+	id uuid NOT NULL,
+	grant_scheme_id int4 NOT NULL,
+    submission_id uuid NULL,
+	name varchar(255) NULL,
+	address_line_1 varchar(255) NULL,
+	address_line_2 varchar(255) NULL,
+	city varchar(255) NULL,
+	county varchar(255) NULL,
+	postcode varchar(255) NULL,
+	org_type varchar(255) NULL,
+	companies_house_number varchar(255) NULL,
+	charity_commission_number varchar(255) NULL,
+	funding_amount varchar(255) NULL,
```

Comment:
Change from DB Design, this should be numeric(16,2)
This is for supporting Spotlight in the future